Renegade X Launcher/MainWindow.xaml.cs

```diff
@@ -445,22 +449,24 @@ private void ServerInfoGrid_SelectionChanged(object sender, SelectionChangedEven
             // Set version mismatch message visibility and join button opacity
             if (VersionCheck.GetGameVersionName() == selected.GameVersion)
             {
-                version_mismatch = false;
+                VersionMismatch = false;
                 SD_VersionMismatch.Visibility = Visibility.Hidden;
                 this.Join_Server_Btn.Background.Opacity = 1.0;
                 this.Join_Server_Btn.Content = "Join Server";
             }
             else
             {
-                version_mismatch = true;
+                VersionMismatch = true;
                 SD_VersionMismatch.Visibility = Visibility.Visible;
                 this.Join_Server_Btn.Background.Opacity = 0.5;
                 this.Join_Server_Btn.Content = "Server Version Mismatch";
             }
-
-            System.Windows.Media.Animation.Storyboard sb = this.FindResource("JoinButtonGlow") as System.Windows.Media.Animation.Storyboard;
-            System.Windows.Media.Animation.Storyboard.SetTarget(sb, this.Join_Server_Btn);
-            sb.Begin();
+            
+            // Ax: This just doesnt work, i assume it was meant to make the Join Game button actually "glow", however it just makes it fade out and back in when a user picks a new server in the list
+            // Schmitz - i assume this was something you were working on?
+            //System.Windows.Media.Animation.Storyboard sb = this.FindResource("JoinButtonGlow") as System.Windows.Media.Animation.Storyboard;

```

Comment:
We had added this to increase visibility of the join button. I agree that it's not the prettiest implementation, but we should make the button do *something* when you click on a server so players know to click it.